src/blocks/scratch3_pen.js

```diff
@@ -3,6 +3,8 @@ const BlockType = require('../extension-support/block-type');
 const Cast = require('../util/cast');
 const Clone = require('../util/clone');
 const Color = require('../util/color');
+const formatMessage = require('format-message');
+const translations = require('../../translations/pen.json');
```

Comment:
This is a temporary way to include translations for testing, not the permanent solution.